monai/transforms/transforms.py

```diff
@@ -877,8 +877,75 @@ def __call__(self, img):
         assert np.all(max_end[:sd] >= self.roi_end[:sd]), 'roi end out of image space.'
 
         slices = [slice(None)] + [slice(s, e) for s, e in zip(self.roi_start[:sd], self.roi_end[:sd])]
-        data = img[tuple(slices)].copy()
-        return data
+        data = img[tuple(slices)]
+        return data.copy() if self.copy else data
+
+
+class CenterSpatialCrop:
+    """
+    Crop at the center of image with specified ROI size.
+
+    Args:
+        roi_size (list, tuple): the spatial size of the crop region e.g. [224,224,128]
+    """
+
+    def __init__(self, roi_size):
+        self.roi_size = roi_size
+
+    def __call__(self, img):
+        center = [i // 2 for i in img.shape[1:]]
+        cropper = SpatialCrop(roi_center=center, roi_size=self.roi_size, copy=False)
+        return cropper(img)
+
+
+class RandCenterSpatialCrop(Randomizable):
```

Comment:
i think the class name should be `RandLocationSpatialCrop` or `RandLocSpatialCrop`, to be consistent with `RandSizeSpatialCrop`

Reply:
changed in the latest patch.
Thanks.